engines/asm/e_padlock-x86_64.pl

```diff
@@ -276,6 +294,115 @@
 	movups	%xmm3,48(%rdx)
 	ret
 .size	padlock_sha512_blocks,.-padlock_sha512_blocks
+.globl  gmi_reload_key
+.type   gmi_reload_key,\@abi-omnipotent
+.align  16
+gmi_reload_key:
+    pushf
```

Comment:
Actually my comment should have been here, of course

Reply:
Yes, this file is for  x86_64 :)

Reply:
Okay, when you step with the gdb debugger here, the stack walk will
not work, so this needs cfi instrumentation for that.

Please look at other sample codes, how to do that.
It may be more easy to solve if this function has a frame pinter

Reply:
As an example I added CFI info to the padlock_verify_context
which is completely untested, maybe that can be used as an
example:

```
diff --git a/engines/asm/e_padlock-x86_64.pl b/engines/asm/e_padlock-x86_64.pl
index 17c98c8..4b70efa 100755
--- a/engines/asm/e_padlock-x86_64.pl
+++ b/engines/asm/e_padlock-x86_64.pl
@@ -124,12 +124,16 @@ padlock_key_bswap:
 .type  padlock_verify_context,\@abi-omnipotent
 .align 16
 padlock_verify_context:
+.cfi_startproc
        mov     $arg1,$ctx
        pushf
+.cfi_def_cfa   %rsp,16
        lea     .Lpadlock_saved_context(%rip),%rax
        call    _padlock_verify_ctx
        lea     8(%rsp),%rsp
+.cfi_def_cfa   %rsp,8
        ret
+.cfi_endproc
 .size  padlock_verify_context,.-padlock_verify_context
 
 .type  _padlock_verify_ctx,\@abi-omnipotent
```

Reply:
Thank you very much. 
I'll study it and then resolve it.